.github/workflows/deploy-lambda.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy Lambda function
+
+on:
+  workflow_call:
+    inputs:
+      function_name:
+        description: "The name of the function to deploy"
+        required: true
+        type: string
+    secrets:
+      AWS_ACCOUNT_ID:
+        description: "The AWS account ID used to determine the ECR registry"
+        required: true
+      AWS_REGION:
+        description: "The AWS region used to determine the ECR registry"
+        required: true
+      AWS_ECR_ACCESS_KEY_ID:
+        description: "The access key ID used to log into AWS ECR"
+        required: true
+      AWS_ECR_SECRET_ACCESS_KEY:
+        description: "The secret access key ID used to log into AWS ECR"
+        required: true
+
+permissions:
+  contents: write
```

Comment:
Just double checking, are these correct? Seems weird that you'd need to write contents to deploy.

Reply:
Nope. I double-checked: https://github.com/exercism/snippet-extractor/actions/runs/2071017622 I've disabled all permissions